tests/backends/test_docker.py

```diff
@@ -54,3 +56,73 @@ def test_custom_master_mounts(
             local_file.write(new_content)
             result = master.run_as_root(args=args)
             assert result.stdout.decode() == new_content
+
+
+class TestBadParameters:
+    """
+    Tests for bad parameters passed to Docker clusters.
+    """
+
+    def test_no_build_artifact(self, tmpdir: local) -> None:
+        """
+        The docker backend requires a build artifact in order
+        to launch a DC/OS cluster.
+        """
+        with pytest.raises(ValueError) as excinfo:
+            with Cluster(
+                cluster_backend=Docker(workspace_dir=tmpdir),
+                build_artifact=None,
+                masters=1,
+                agents=0,
+                public_agents=0,
+            ):
+                pass  # pragma: no cover
+
+        expected_error = (
+            'The Docker backend only supports creating new clusters. '
```

Comment:
Instead, make it really clear what a user should do. Perhaps "The Docker backend only supports creating new clusters. `build_artifact` must be a path to a build artifact."